uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -17,285 +17,601 @@
+                                        font-weight: inherit;
+                                        text-decoration: none;
+                                        padding: 8px 24px;
+                                        width:100%;
+                                        min-width: 100%;
+
+                                        &.disabled {
+                                            pointer-events: none;
+                                            cursor: default;
+                                        }
+
+                                        &:hover, &:focus {
+                                            background: @navbar-item-hover-background-color;
+                                            color: @navbar-item-hover-text-color;
+
+                                            i {
+                                                opacity: 1;
+                                            }
+                                        }
+
+                                        i {
+                                            float: right;
+                                            margin-top: 2px;
+                                            margin-right: -6px;
+                                            opacity: .5;
+                                        }
+
+                                    }
+
+                                    &.portal-navigation-gripper, &.portal-navigation-delete {
+                                        display: none;
+                                    }
+
+                                    &.dropdown-toggle.portal-navigation-dropdown {
+                                        display: inline-block;
+                                        position: relative;
+                                        width: 17px;
+                                        top: -5px;
+
+                                        &:hover, &:focus, &:active {
+                                            background-color: transparent;
+                                        }
+
+                                        &:focus {
+                                            outline: dotted 1px;
+                                        }
+
+                                        .caret {
+                                            margin: 10px 7px;
+                                            -webkit-transform: rotate(-90deg);
+                                            -moz-transform: rotate(-90deg);
+                                            -o-transform: rotate(-90deg);
+                                            -ms-transform: rotate(-90deg);
+                                            transform: rotate(-90deg);
+                                            -webkit-transition: transform 150ms ease-out;
+                                            -moz-transition: transform 150ms ease-out;
+                                            -o-transition: transform 150ms ease-out;
```

Comment:
not for this PR but it the future it would be nice to use [autoprefixer](https://github.com/postcss/autoprefixer) to simplify browser support.